src/components/braintree/utils.ts

```diff
@@ -97,3 +98,22 @@ export const getBraintreeNamespace = (
         loadCustomScript({ url: BRAINTREE_PAYPAL_CHECKOUT_SOURCE }),
     ]).then(() => getBraintreeWindowNamespace());
 };
+
+/**
+ * Get the merchantId from the source list
+ *
+ * @param {Array|string} source - the source list with merchant identifiers
+ * @throws {Error} when the merchant list has more than 1 value
+ * @returns {string|undefined} the merchantId or an undefined value
+ */
+export const getMerchantId = (
+    source?: Array<string> | string
+): string | undefined => {
+    const isSourceArray = Array.isArray(source);
+
+    if (isSourceArray && source.length > 1) {
+        throw new Error(BRAINTREE_MULTIPLE_MERCHANT_IDS);
+    }
+
+    return isSourceArray ? (source || [])[0] : source;
```

Comment:
Can you use toString() to take care of converting an array of size 1 or a string to a string?
```suggestion
    return source.toString();
```